src/predict_3dpose.py

```diff
@@ -75,12 +75,19 @@
   'batch_normalization' if FLAGS.batch_norm else 'no_batch_normalization',
   'use_stacked_hourglass' if FLAGS.use_sh else 'not_stacked_hourglass',
   'predict_14' if FLAGS.predict_14 else 'predict_17')
-
+train_dir='train'
 print( train_dir )
 summaries_dir = os.path.join( train_dir, "log" ) # Directory for TB summaries
 
 # To avoid race conditions: https://github.com/tensorflow/tensorflow/issues/7448
-os.system('mkdir -p {}'.format(summaries_dir))
+def mkdir_p(path):
+  try:
+    if not os.path.isdir(path):
+      os.makedirs(path)
+  except OSError as exc:  # Python >2.5
+    raise exec
+#os.system('mkdir -p {}'.format(summaries_dir))
+mkdir_p(summaries_dir)
```

Comment:
I see how this could be annoying for windows users. May I suggest that we simply test for OS and, if it is windows, skip this line?

Reply:
Where is "this line"?
If that is "except ... raise exec",I think that line is unnecessary too.

Reply:
I mean that, instead of `os.system('mkdir -p {}'.format(summaries_dir))`
we could have something that tests whether the machine is running non-unix, and if that is the case then just skip the line -- tensorflow should create the directory and, if it crashes, it's tf's fault, but at least it will not crash on windows *every time*.